src/less/calendar/calendar.less

```diff
@@ -0,0 +1,180 @@
+@import "../variables/variables.less";
```

Comment:
I'm not seeing any responsive adjustments and the UI of the multi-month calendars on narrower screens look broken. 
<img width="488" alt="image" src="https://user-images.githubusercontent.com/1675667/222212475-f3a0ecbb-2b36-46c8-8f14-202006fa7036.png">

<img width="519" alt="image" src="https://user-images.githubusercontent.com/1675667/222212848-3facb443-0d08-4a5b-aa29-f8e8fbf33a46.png">

Maybe on narrower screens we only show one header and calendar? Eventually it would take `js` to shuffle through the months, but we should probably have our `css` strategy in place now. Some of it may also depend on how we're going to be dealing with this in ebay ui.

Reply:
We've talked about this before. There are a few issues with hiding stuff via CSS, but the primary problem is that without adding more markup specifically for this purpose there will be inconsistencies for tabindex and screen reader labels. I will add some documentation here which states that applications should limit the number of visible months with JavaScript.

Reply:
Discussed offline, this should be resvolved.